pages/index.css

```diff
@@ -2,3 +2,12 @@
 @import url(../vendor/fonts/fonts.css);
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
+
+@import url(../blocks/research/research.css);
```

Comment:
@Woolfs1423 Let's organize our `index.css` for better clarity. Please separate each block with one empty line and add a comment before each one.

You can refer to how Serezha and I did that [here](https://github.com/p-foundation/pink-app/blob/main/pages/index.css)